### PR TITLE
feat(AutoModeration): v13 support `customMessage` 

### PR DESCRIPTION
**Pull request by jaw0r3k**
**Please describe the changes this PR makes and why it should be merged:**
 Backports: 
 1. #9171

**Status and versioning classification:**
- Code changes have been tested against the Discord API
- I know how to update typings and have done so
- This PR changes the library's interface (methods or parameters added)

### CODE REVIEW

**Review by vladfrangu — APPROVED** [review]

File: src/managers/AutoModerationRuleManager.js [hunk 1]
```diff
@@ -74,6 +74,8 @@ class AutoModerationRuleManager extends CachedManager {
    * @typedef {Object} AutoModerationActionMetadataOptions
    * @property {GuildTextChannelResolvable|ThreadChannel} [channel] The channel to which content will be logged
    * @property {number} [durationSeconds] The timeout duration in seconds
+   * @property {string} [customMessage]
+   * The custom message that will be shown to members whenever the rule blocks their message.
```
Comment (Jiralite) [inline src/managers/AutoModerationRuleManager.js:78]: This description doesn't match the string in main.

File: src/structures/AutoModerationRule.js [hunk 2]
```diff
@@ -101,6 +101,8 @@ class AutoModerationRule extends Base {
        * @typedef {Object} AutoModerationActionMetadata
        * @property {?Snowflake} channelId The id of the channel to which content will be logged
        * @property {?number} durationSeconds The timeout duration in seconds
+       * @property {?string} customMessage
+       * The custom message that will be shown to members whenever the rule blocks their message.
```
Comment (Jiralite) [inline src/structures/AutoModerationRule.js:105]: This description doesn't match the string in main.

**Review by Jiralite — CHANGES_REQUESTED** [review]

File: src/managers/AutoModerationRuleManager.js (hunk 1 above)
Comment (jaw0r3k) [reply src/managers/AutoModerationRuleManager.js:78]: it was copied from ddevs as i remember

Comment (Jiralite) [reply src/managers/AutoModerationRuleManager.js:78]: You shouldn't be doing that, especially if you call this pull request a backport.

**Review by kyranet — APPROVED** [review]

**Pull request merged**